control-plane/pkg/security/secrets_provider_legacy_channel_secret.go

```diff
@@ -39,16 +39,16 @@ func ResolveAuthContextFromLegacySecret(s *corev1.Secret) (*NetSpecAuthContext,
 
 	virtualSecret := s.DeepCopy()
 	virtualSecret.Data[ProtocolKey] = []byte(protocolStr)
-	if v, ok := virtualSecret.Data["sasltype"]; ok {
+	if v, ok := virtualSecret.Data[SaslType]; ok {
 		virtualSecret.Data[SaslMechanismKey] = v
 	}
-	if v, ok := virtualSecret.Data["saslType"]; ok {
+	if v, ok := virtualSecret.Data[SaslTypeLegacy]; ok {
```

Comment:
when can we remove this legacy?

Reply:
Maybe we can send an email + slack update and see if anyone is using currently?

Reply:
OK. that can be done. but is a different issue